docs/dev/menu_hierarchies.md

```diff
@@ -1788,3 +1788,7 @@ The Audio Clip menu contains the following menu hierarchy:
 </details>
 
 </details>
+
+## Additional Information
```

Comment:
This should not move to dev. This is a user facing document to help them understand the menu hierarchies.

Reply:
Right.  So I'm not familiar with the process of reverting this change from this PR.  Should I just move the file back to root?

Reply:
> Right. So I'm not familiar with the process of reverting this change from this PR. Should I just move the file back to root?

yes you would just move the file back

I would also remove the developer note that was added at the bottom

then you can commit the changes